bindings/java/iota-wallet-java/src/main/java/org/iota/types/secret/MnemonicSecretManager.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package org.iota.types.secret;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.google.gson.annotations.JsonAdapter;
+
+import java.lang.reflect.Type;
+
+@JsonAdapter(MnemonicSecretManager.MnemonicSecretManagerAdapter.class)
+public class MnemonicSecretManager extends SecretManager {
```

Comment:
Can you add a comment similar to here https://github.com/iotaledger/iota.rs/commit/7df2da814c399616282f8860e6010523255c5f93 ?

Reply:
added